### PR TITLE
GSoD Case study

### PR DESCRIPTION
**Pull request by JyeSmith**

### CODE REVIEW

**Review by deadbytefpv — APPROVED** [review]

File: docs/gsod.md [hunk 1]
```diff
@@ -72,3 +72,76 @@ ELRS has set its next major release goal before November to include release note
...
+
+A review of the existing user documentation by a technical writer, who has not previously been exposed to FPV, will bring a fresh pair of eyes to the project and help identify knowledge gaps.
+
+**Project Description**
+
+**Creating the proposal**
+
+The proposal idea was pretty easy for us… as we like to keep it simple.  GSoD is about documents, and hiring a Technical Writer, so we rubbed our 2 brain cells together and decided we needed help reviewing a writing docs.  The exploding ELRS users base meant we can not continue to provide 1 on 1 support to new users, unlike in the beginning of the project.  The demand for more and better documents is and will be ongoing during the ELRS growth period.  However, the community appears to have reached a critical mass and the user base is great at supporting new users.
```
Comment (wvarty) [inline docs/gsod.md:98]: Is this supposed to be "decided we needed help reviewing **and** writing docs"?

File: docs/gsod.md [hunk 2]
```diff
@@ -72,3 +72,76 @@ ELRS has set its next major release goal before November to include release note
...
+
+$1,849.42 of the proposed $2000 hardware budget has been spent.
+
+$600 of the proposed $1000 Volunteer stipends has been claimed.
+
+The project was able to stay within the initial budget.  Some ExpressLRS hardware manufacturers were even able to help with additional hardware and we thank (in alphabetical order) [BETAFPV](https://betafpv.com/), [Happymodel](https://www.happymodel.cn/), and [Radiomaster](https://www.radiomasterrc.com/) for their assistance in providing hundreds of dollars of free hardware.
+
+**Participants**
+
+The project had 5 volunteers that were either Developers or long term advance users on Discord and known to ExpressLRS…  deadbyte, maybenikhil, StonedDawg, braincell power user, and commanderguy.  Our Technical Writer AghaSaad was found through reviewing resumes.
```
Comment (wvarty) [inline docs/gsod.md:114]: Seems this should be "long term advance**d** users"

File: docs/gsod.md [hunk 3]
```diff
@@ -72,3 +72,76 @@ ELRS has set its next major release goal before November to include release note
...
+
+While no one dropped out of the project, there were personal issues that arose during the period that made it difficult for some volunteers to be available.  This did impact the project's timeline and delay hardware delivery.
+
+Selecting a Writer with a pre-existing interest in RC or similar hobby that aligned with the projects, and not focus solely on writing experience, may also have assisted with keeping to the timeline.
+
+**Timeline**
+
+Our initial timeline has been heavily delayed through the delivery of hardwar to our Writer… and then the eventual confiscation of said hardware.  More details below.
```
Comment (wvarty) [inline docs/gsod.md:122]: Missing "e" on hardware

File: docs/gsod.md (hunk 1 above)
Comment (maybenikhil) [reply docs/gsod.md:98]: rewriting, making a bit more professional

File: docs/gsod.md [hunk 4]
```diff
@@ -95,7 +95,7 @@ A review of the existing user documentation by a technical writer, who has not p
 
 **Creating the proposal**
 
-The proposal idea was pretty easy for us… as we like to keep it simple.  GSoD is about documents, and hiring a Technical Writer, so we rubbed our 2 brain cells together and decided we needed help reviewing a writing docs.  The exploding ELRS users base meant we can not continue to provide 1 on 1 support to new users, unlike in the beginning of the project.  The demand for more and better documents is and will be ongoing during the ELRS growth period.  However, the community appears to have reached a critical mass and the user base is great at supporting new users.
+The proposal idea was pretty easy for us… as we like to keep it simple.  GSoD is about documents, and hiring a Technical Writer, so we rubbed our 2 brain cells together and decided we needed help reviewing a writing docs.  The exploding ELRS users base meant we can not continue to provide 1 on 1 support to every users, unlike in the beginning of the project.  The demand for more and better documents is and will be ongoing during the ELRS growth period.  However, the community appears to have reached a critical mass and the user base is great at supporting new users.
```
Comment (wvarty) [inline docs/gsod.md:98]: Should be:
"The exploding ELRS user base meant we can not continue to provide 1 on 1 support to every user."

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**